src/pages/components/TodoList/components/TodoItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { MdDelete } from "react-icons/md";
+import { Todo } from "src/types";
```

Comment:
Since the Types are related to TODO type, let's create a type.ts file in the component itself not in the generic app types